tests/unit/introspection-2.tcl

```diff
@@ -76,4 +76,18 @@ start_server {tags {"introspection"}} {
         assert_match {*calls=1,*} [cmdstat expire]
         assert_match {*calls=1,*} [cmdstat geoadd]
     } {} {needs:config-resetstat}
+
+    test {COMMAND GETKEYS MEMORY USAGE} {
+        assert_equal {key} [r command getkeys memory usage key]
+    }
```

Comment:
do we have a test for COMMAND GETKEYS for a command with sub-commands (other than MEMORY USAGE)?
maybe test some basics like GET and and advance one like XGROUP